api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/VisitTranslatorImplTest.java

```diff
@@ -290,4 +333,25 @@ public void toFhirResource_shouldTranslateTypeToEncounterType() {
 		assertThat(result.getType(), not(empty()));
 		assertThat(result.getTypeFirstRep(), equalTo(codeableConcept));
 	}
+	
+	@Test
+	public void toFhirResource_shouldTranslateDateStartedStoppedToPeriod() {
+		Visit visit = new Visit();
+		visit.setUuid(VISIT_UUID);
+		
+		Period period = new Period();
+		period.setStart(periodStart);
+		period.setEnd(periodEnd);
+		
+		visit.setStartDatetime(periodStart);
+		visit.setStartDatetime(periodEnd);
+		
+		when(visitPeriodTranslator.toFhirResource(ArgumentMatchers.any())).thenReturn(period);
+		
+		Encounter result = visitTranslator.toFhirResource(visit);
+		
```

Comment:
@pmanko  thanks for working on this,add a ticket id to your commit and you can also remove the spaces from your code